src/components/Vega.svelte

```diff
@@ -209,6 +243,14 @@
     }
   });
 
+  const debouncedResize = debounce(() => {
```

Comment:
one could consider debouncing it not only within a vega component but even within all vega components by moving it to a separate file. Thus, it would be like a static for this component

Reply:
Yes, I was thinking about something like that, and we should do it, but I didn't want to possibly cause other problems.

Actually, throttling might make more sense here, since every resize event is identical, and every afterUpdate event is identical, so it is only the rate that should matter.  But debouncing effectively defers all resizing until the mouse stops moving, in the case of the mouse motion triggering tooltip display that triggers afterUpdate events.  

So we should really figure out whether we need the afterUpdate event after the tooltip display (and one event for each chart!) and maybe figure out how to catch that and throw it away.